frontend/src/features/apartment/ApartmentImprovementsPage.tsx

```diff
@@ -39,7 +39,6 @@ const ImprovementAddLineButton = ({onClick}) => {
         <Button
             onClick={onClick}
             iconLeft={<IconPlus />}
-            variant="secondary"
```

Comment:
On Housing Company page the button has `variant="secondary"`, so it's not filled black, where as on Apartment page the button is black.